Index: src/components/SideCart.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/components/SideCart.js	(date 1558704201143)
+++ src/components/SideCart.js	(date 1558704201143)
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const SideCart = () => {
+  return (
+      <div>
+        hello
+      </div>
+  );
+};
+
+export default SideCart;
\ No newline at end of file
